### ISSUE TITLE
variant: 'outlined' gives React error

### ISSUE THREAD

**Issue by octopus-travels** [KevinVandy/material-react-table#878]
### material-react-table version

2.0.6

### react & react-dom versions

18.2

### Describe the bug and the steps to reproduce it

When you add the prop muiEditTextFieldProps: {variant: 'outlined'} to a column and it gives JS error. Works fine with variant: 'filled' and variant: 'standard'.

### Minimal, Reproducible Example - (Optional, but Recommended)

```
      {
        accessorKey: 'firstName',
        header: 'First Name',
        muiEditTextFieldProps: {
          variant: 'outlined',
          type: 'email',
          required: true,
          error: !!validationErrors?.firstName,
          helperText: validationErrors?.firstName,
          //remove any previous validation errors when user focuses on the input
          onFocus: () =>
            setValidationErrors({
              ...validationErrors,
              firstName: undefined,
            }),
          //optionally add validation checking for onBlur or onChange
        },
```

### Screenshots or Videos (Optional)

![Screenshot_2023-11-28_at_22 18 43](https://github.com/KevinVandy/material-react-table/assets/114671390/8ec2002d-08df-4d05-8e15-f7038546e68e)

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment]
fixed in v2.1.0

**Comment by octopus-travels** [comment]
Thank a lot for fixing it
